sw/device/silicon_creator/rom/e2e/BUILD

```diff
@@ -3920,36 +3920,46 @@ test_suite(
 opentitan_functest(
     name = "presigned_fpga_binary",
     cw310 = cw310_params(
-        bitstream = ":bitstream_sigverify_spx_prod_enabled_zero",
+        bitstream = "//hw/bitstream:rom_with_real_keys_otp_test_unlocked0",
     ),
-    ot_flash_binary = "//sw/device/silicon_creator/rom/e2e/presigned_images:presigned",
+    ot_flash_binary = "//sw/device/silicon_creator/rom/e2e/presigned_images:empty_test",
     signed = False,
     targets = ["cw310_rom_with_real_keys"],
 )
 
-# TODO(#18686): update to be pres-signed
-opentitan_functest(
+opentitan_flash_binary(
```

Comment:
this target is just for building binary to be signed, right? This is a minor point but wouldn't it be nice to be able to run this test with fake keys before signing the binary?

Reply:
We'd need another copy of this test: the golden hash encoded in the test source code is for `rom_with_real_keys`.